Marsiple/App/Todos/TodosViewController.swift

```diff
@@ -80,9 +100,32 @@ private extension TodosViewController {
         }
     }
     
+    func setupNavigationBar() {
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addButtonTapped))
+    }
+    
+    // NOTE: This is just temporary, until network is implemented
     func setupTodos() -> [[Todo]] {
-        let uncompleted = [Todo](repeating: Todo(id: 1, title: "Todo 1", completed: false, userId: 1), count: 10)
-        let completed = [Todo](repeating: Todo(id: 1, title: "Todo 2", completed: true, userId: 1), count: 10)
+        var uncompleted = [Todo]()
+        var completed = [Todo]()
+        for i in 0...9 {
+            uncompleted.append(Todo(id: i, title: "Todo \(0)\(i)", completed: false, userId: -1))
+            completed.append(Todo(id: i, title: "Todo \(1)\(i)", completed: true, userId: -1))
+        }
         return [uncompleted, completed]
     }
+    
+    func reloadData() {
```

Comment:
This is the `Wild West` approach that I mentioned earlier. 

I call it like that because, what you do here is that you attempt to restore to a stable state using a set of rules (filtering todos, to get the correct state). I ask you this, why should a `completed` todo `EVER END UP` in the `active` section? That's invalid logic right at the start.

The `todos` should be modified by a concise and isolated set of operations, that, once completed, are fully finished - both on the `UI` and `Business logic` side.